src/components/Login/Login.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef, useState } from 'react'
+import { Grid,Paper, Avatar, TextField, Button, Typography,Link } from '@material-ui/core'
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import { useAuth } from '../Contexts/AuthContext';
+import { useNavigate } from 'react-router-dom';
+import { Alert } from '@mui/material'
+
+
+const Login = ({ handleChange }) => {
+
+	const paperStyle={padding : 20, height:'60vh', width:300, margin:"0 auto"}
```

Comment:
Do you think this stuff would be better in a Login.css file? I know it's not a lot of styling, but we may add more to these classes once we have our UI design